src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -206,6 +206,54 @@ export default ({ req, res, log, error }: any) => {
   });
 };
 ```
+```go
+package handler
+
+import (
+	"fmt"
+	"os"
+
```

Comment:
Is this space here standard formatting for go?

Reply:
Yes, this formatting is standard for go. Please take a look at the @byawitz's [playground-go ](https://github.com/appwrite/playground-for-go/pull/3/files#diff-fcc82ca569b20f8e21654d678d2eae96e86a953aeff6c87b1a0200aa2944bf05), where you can see the same space formatting there too.